git/cmd.py

```diff
@@ -695,15 +695,14 @@ def __iter__(self) -> "Git.CatFileContentStream":
             return self
 
         def __next__(self) -> bytes:
-            return next(self)
-
-        def next(self) -> bytes:
             line = self.readline()
             if not line:
                 raise StopIteration
 
             return line
 
+        next = __next__
+
```

Comment:
Feel free to commit through the github UI if you want to drop this Python2 compatibility layer entirely:
```suggestion
```

Note that there may be some downstream code that relies on using `next` because `__next__` was broken though!

Reply:
Ah, right. Oh well, let's be conservative then.